src/map/script.c

```diff
@@ -20698,7 +20724,7 @@ static BUILDIN(getvariableofpc)
 	}
 
 	if (!sd->regs.vars)
-		sd->regs.vars = i64db_alloc(DB_OPT_RELEASE_DATA);
+		sd->regs.vars = i64db_alloc(DB_OPT_BASE);
```

Comment:
is this is memory leak?

Reply:
nope, and this is exactly what is used in pc.c too